devel/smake/Portfile

```
@@ -1,41 +1,116 @@
+                        ${worksrcpath}/TEMPLATES/Defaults.xcc
 }
 
-livecheck.url       https://sourceforge.net/projects/s-make/files/
-livecheck.regex     ${name}-(\\d+(?:\\.\\d+)*)${extract.suffix}
+proc port_conflict_check {p_port_name p_conflict_ver_min p_conflict_ver_max} {
+    ui_info "Checking for conflict against port: ${p_port_name}"
+
+    if { ![catch {set port_conflict_ver_info [lindex [registry_active ${p_port_name}] 0]}] } {
+        set port_conflict_ver [lindex ${port_conflict_ver_info} 1]_[lindex ${port_conflict_ver_info} 2]
+        ui_info "${p_port_name} active version: ${port_conflict_ver}"
+
+        if { [vercmp ${port_conflict_ver} ${p_conflict_ver_min}] >= 0
+            && [vercmp ${port_conflict_ver} ${p_conflict_ver_max}] <= 0 } {
+
+            ui_info "${p_port_name} conflicts; declare build conflict"
+            conflicts_build-append \
+                ${p_port_name}
+        }
+    }
+}
+
+port_conflict_check     cdrtools 0.0 3.01_1
+
+# https://trac.macports.org/ticket/31616
+compiler.blacklist-append\
+                        llvm-gcc-4.2 macports-llvm-gcc-4.2 {clang < 300}
+
+configure.ldflags-append -lintl
+
+depends_build-append    port:gmake \
+                        port:gettext
+                        
+depends_lib-append      port:gettext-runtime
+
+build.type              gnu
+                        
+build.args-append       INS_BASE="${destroot}${prefix}" \
+                        INS_RBASE="${destroot}${prefix}" \
+                        DEFOSINCDIRS="${prefix}/include" \
+                        CC_OPT=${configure.optflags}
+
+destroot.destdir        INS_BASE="${destroot}${prefix}" \
+                        INS_RBASE="${destroot}${prefix}" \
+                        MANDIR=man \
+                        DEFINSUSR=${install.user} \
+                        DEFINSGRP=${install.group}
+
+# NOTE:  This port first builds a bootstrapped version of smake using gmake.
+#        The final version of smake (and its static associated libraries) are built using
+#        the boostrapped version of smake which does NOT support parallel building.
+#        i.e.  the j flag is ignored by the boostrapped version of smake and
+#              the final version of smake.
+use_parallel_build      no
+
+destroot {
+                        xinstall -m 0755    {*}[glob ${worksrcpath}/smake/OBJ/*/smake] \
+                        ${destroot}${prefix}/bin
+                        
+                        xinstall -m 0644    {*}[glob ${worksrcpath}/smake/OBJ/*/*/smake.1] \
+                        ${destroot}${prefix}/share/man/man1
+                        
+                        xinstall -m 0644    {*}[glob ${worksrcpath}/man/man5/OBJ/*/*/makefiles.5] \
+                        ${destroot}${prefix}/share/man/man5
+                        
+                        xinstall -m 0644    {*}[glob ${worksrcpath}/man/man5/OBJ/*/*/makerules.5] \
+                        ${destroot}${prefix}/share/man/man5
```

```suggestion
                        xinstall -m 0644    {*}[glob ${worksrcpath}/man/man5/OBJ/*/*/makefiles.5] \
                        {*}[glob ${worksrcpath}/man/man5/OBJ/*/*/makerules.5] \
                        ${destroot}${prefix}/share/man/man5
```
you may instal more than one file via one call to the same folder I guess

---

Yes.  That works!  I forgot how smart `xinstall` really is.  I will make the change and force push it.  Give me a minute.